components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -44,22 +45,32 @@ const ContributorProfileTab = ({
   const { data: highlights, isError, isLoading } = useFetchUserHighlights(login || "");
   const [inputVisible, setInputVisible] = useState(false);
 
+  const hasHighlights = highlights?.length > 0;
+  const router = useRouter();
+
+  const handleTabUrl = (tab: string) => {
+    router.push(`/user/${login}/${tab.toLowerCase()}`);
+  };
+
   useEffect(() => {
     setInputVisible(highlights && highlights.length !== 0 ? true : false);
+    if (!hasHighlights) router.push(`/user/${login}/contributions`);
```

Comment:
we should add an else statement here to  push to `/user/${login}/highlights`

Reply:
Absolutely, thanks 👍